crates/cheatcodes/spec/src/vm.rs

```diff
@@ -1077,6 +1077,22 @@ interface Vm {
     #[cheatcode(group = String)]
     function parseBool(string calldata stringifiedValue) external pure returns (bool parsedValue);
 
+    /// Converts the given `string` value to Lowercase.
+    #[cheatcode(group = String)]
+    function toLowercase(string calldata inputString) external pure returns (string memory outputString);
+    /// Converts the given `string` value to Uppercase.
+    #[cheatcode(group = String)]
+    function toUppercase(string calldata inputString) external pure returns (string memory outputString);
+    /// Trims leading and trailing whitespace from the given `string` value.
+    #[cheatcode(group = String)]
+    function trim(string calldata inputString) external pure returns (string memory outputString);
+    /// Replaces occurrences of `from` in the given `string` with `to`.
+    #[cheatcode(group = String)]
+    function replace(string calldata inputString, string calldata from, string calldata to) external pure returns (string memory outputString);
+    /// Splits the given `string` into an array of strings divided by the `delimiter`.
+    #[cheatcode(group = String)]
+    function split(string calldata inputString, string calldata delimiter) external pure returns (string[] memory outputStrings);
+
```

Comment:
style nit—can just be `input` rather than `inputString`—the type is implied already

Reply:
should I use inputStr? or just input?

Reply:
@Evalir ?

Reply:
`input` is enough, as suggested by @mds1 :)